store/multiversion/mvkv.go

```diff
@@ -359,18 +360,20 @@ func (store *VersionIndexedStore) UpdateReadSet(key []byte, value []byte) {
 
 // Write implements types.CacheWrap so this store can exist on the cache multi store
 func (store *VersionIndexedStore) Write() {
-	//TODO implement me
-	panic("implement me")
+	panic("not implemented")
 }
 
 // GetEvents implements types.CacheWrap so this store can exist on the cache multi store
 func (store *VersionIndexedStore) GetEvents() []abci.Event {
-	//TODO implement me
-	panic("implement me")
+	panic("not implemented")
 }
 
 // ResetEvents implements types.CacheWrap so this store can exist on the cache multi store
 func (store *VersionIndexedStore) ResetEvents() {
-	//TODO implement me
-	panic("implement me")
+	panic("not implemented")
+}
+
+func (store *VersionIndexedStore) UpdateIterateSet(iterationTracker iterationTracker) {
```

Comment:
wasnt this already present from occ-main, or is the diff highlight just weird because of merge conflict resolution

Reply:
This line doesn't seem to show up in the overall diff, just in the commit itself (a merge from occ-main).